src/networkremote/incomingmsg.cpp

```diff
@@ -0,0 +1,43 @@
+#include "incomingmsg.h"
+#include <QTcpSocket>
```

Comment:
Place system includes before headers in strawberry

Reply:
Done